lib/pgp.ml

```diff
@@ -0,0 +1,677 @@
+let ( >>= ) = Result.bind
```

Comment:
We talked about it with @bbc2 and the reason we didn't use containers when the file was written was to not introduce a big dependency in this library : since it is open-source, we don't really know who uses it. As you said : to be discussed later.

Reply:
That is reasonable. I guess it means we ditch compatibility with 4.07 (which should be fine).